CMakeLists.txt

```diff
@@ -16,13 +16,16 @@ project(vikunja
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED 14)
 
-include(CTest)
+include(CMakeDependentOption)
+include(FetchContent)
 
 option(vikunja_BUILD_EXAMPLES "Build the examples" OFF)
 option(vikunja_BUILD_DOXYGEN "Build the doxygen documentation" OFF)
 option(vikunja_ENABLE_EXTRA_WARING "Enable extra warnings" ON)
 option(VIKUNJA_REDUCE_COMPARING_BENCHMARKS_ENABLE "Should the reduce integration test run some comparison benchmarks?" OFF)
 option(VIKUNJA_TRANSFORM_COMPARING_BENCHMARKS_ENABLE "Should the transform integration test run some comparison benchmarks?" OFF)
+option(BUILD_TESTING "Build the testing tree." OFF)
```

Comment:
Do you need to define `BUILD_TESTING`? I thought it is automatically defined by `include(CTest)`, which I think you should do here.

Reply:
resolved here: https://github.com/alpaka-group/vikunja/pull/35#discussion_r726922630